packages/client/tests/functional/jest.config.js

```diff
@@ -25,4 +25,22 @@ module.exports = {
   setupFilesAfterEnv: ['./_utils/setupFilesAfterEnv.ts'],
   testTimeout: 10000,
   collectCoverage: process.env.CI ? true : false,
+
+  /**
+   * Limit concurrency to 1 worker for now for two reasons.
+   *
+   * First reason:
+   *
+   * Database isolation in setupTestSuiteDbURI() doesn't seem to work properly.
+   * Tests are using to the same database concurrently, which causes constraint
+   * violation errors and "database does not exist" errors.
+   * TODO: fix this.
```

Comment:
Seems like this is a high priority. Thanks for reporting!

Reply:
I created an issue for this: https://github.com/prisma/prisma/issues/13186